packages/amplify-provider-awscloudformation/src/graphql-transformer-factory/transformer-version.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  $TSContext,
+  pathManager,
+  stateManager,
+} from 'amplify-cli-core';
+import { FeatureFlags } from 'amplify-cli-core';
+
+/**
+ * Shorthand for Feature flag retrieval.
+ */
+const useExperimentalPipelinedTransformerFF = () => FeatureFlags.getBoolean('graphQLTransformer.useExperimentalPipelinedTransformer')
```

Comment:
Could this potentially be an issue? Since this doesn't happen at function call time the loading order of packages could mean the `FeatureFlags` class not being initialized, or if we change a feature flag value during the execution (such as v1 -> v2 migration) these values would be out of date

Reply:
oh nevermind I missed that these are written as a function rather than just resolved values